feature-group-definitions/subgrid.yml

```diff
@@ -1,5 +1,9 @@
 spec: https://drafts.csswg.org/css-grid-2/#subgrids
 caniuse: css-subgrid
+status:
+  is_baseline: false
+  support:
+    firefox: "71"
```

Comment:
It's also in Safari.

Reply:
I have no idea how I missed that. Fixing.